ui/components/app/collectible-default-image/collectible-default-image.stories.js

```diff
@@ -39,4 +43,7 @@ export const HandleImageClick = (args) => (
 HandleImageClick.args = {
   // eslint-disable-next-line no-alert
   handleImageClick: () => window.alert('CollectibleDefaultImage clicked!'),
+  handleImageKeyPress: () =>
```

Comment:
We wont need this when we switch to button.

Reply:
Removed